pepper-apis/src/main/java/org/broadinstitute/ddp/route/GetActivityInstanceRoute.java

```diff
@@ -108,6 +112,16 @@ public Object handle(Request request, Response response) {
             Long languageCodeId = preferredUserLanguage.getId();
             return validateActivityInstance(handle, activityInstance, userGuid, languageCodeId);
         });
+
+        StudySettings studySettings = GoogleAnalyticsMetricsTracker.getStudySettingByStudyGuid(studyGuid);
+        if (studySettings != null && studySettings.isAnalyticsEnabled()) {
+            String gaEventLabel = studyGuid.join(":", GoogleAnalyticsMetrics.EVENT_LABEL_ACTIVITY_INSTANCE);
+            EventHit eventHit = new EventHit(GoogleAnalyticsMetrics.EVENT_CATEGORY_ACTIVITY_INSTANCE,
+                    GoogleAnalyticsMetrics.EVENT_ACTION_ACTIVITY_INSTANCE, gaEventLabel, 1);
+            GoogleAnalyticsMetricsTracker.sendEventMetrics(studyGuid, eventHit);
+        }
+
```

Comment:
Will be nice if there is a convenience method we can call in one-line instead of having to repeat this whole `if` block in a bunch of places.